StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -347,6 +347,9 @@ extension PaymentSheet {
 
         /// The customer's phone number without formatting (e.g. 5551234567)
         public var phone: String?
+
+        /// Initializes billing details
+        public init() { }
```

Comment:
I think here we should generate the default member-wise initializer, wdyt

Reply:
Ooh, forgot that button existed, done